src/main/java/org/example/AppConfig.java

```diff
@@ -0,0 +1,18 @@
+package org.example;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+
+@Configuration
+@PropertySource("classpath:properties")
+public class AppConfig {
```

Comment:
nice idea, but delete the whole class, i don't want the application to keep a user session active to validate the user, this is because a car can have multiple profiles that can be activated when the car is started, so there is no real need for a user session to be active, we just want to have a list of users in the code which we can check against the incoming requests to validate if the user making the request is one of the users that exist in the car profiles